src/sdk/model/o365/o365-restore-session.ts

```diff
@@ -1003,6 +1005,53 @@ export class O365RestoreSession extends Entity {
     return Iland.getHttp().post(`/o365-restore-sessions/${this.uuid}/teams/${teamId}/channels/${channelId}/files/action/restore-files`, request.json);
   }
 
+  /**
+   * Download all O365 Team Files as a zip
+   * @param {string} teamId
+   * @param {string} channelId
+   * @param {function} downloadProgressCallback optional progress callback if user wishes to receive progress events
+   * @returns {Promise<Blob>} will return a promise with the Blob to download
+   */
+  /* istanbul ignore next: autogenerated */
+  async saveAllTeamFiles(teamId: string, channelId: string,
+                         downloadProgressCallback?: (e: any) => void): Promise<Blob> {
+    // tslint:disable-next-line:max-line-length
+    return Iland.getHttp().post(`/o365-restore-sessions/${this.uuid}/teams/${teamId}/channels/${channelId}/files/action/save-all`,
+        null, {
+          headers: {
+            'Accept': Http.versionMime('application/octet-stream')
+          },
+          responseType: 'arraybuffer',
+          onDownloadProgress: downloadProgressCallback
+        }).then((response) => {
+          return new Blob([response.data], { type: 'application/zip' });
+        });
+  }
+
+  /**
+   * Download a O365 Team file as a zip
+   * @param {string} teamId
+   * @param {string} channelId
+   * @param {O365TeamItemsExportRequest} request
+   * @param {function} downloadProgressCallback optional progress callback if user wishes to receive progress events
+   * @returns {Promise<Blob>} will return a promise with the Blob to download
+   */
+  /* istanbul ignore next: autogenerated */
+  async saveTeamFile(teamId: string, channelId: string, request: O365TeamItemsExportRequest,
```

Comment:
this endpoint is capable of saving multiple files since it takes an array of item ids no?

Reply:
Yes this one and the other methods similar to this is capable of saving / exporting multiple, I updated the documentation for the methods to better show that, not sure if you had maybe better method names in mind or if it can be kept the same?

Reply:
Yup just tested and everything works as expected for multiple files, went ahead and added the 's' to the method names and docs are updated 👍🏼

Reply:
thank you for that @Juchazari !!